apps/etterlatte-saksbehandling-ui/client/src/components/klage/vurdering/KlageInitiellVurdering.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Heading, Radio, RadioGroup, Textarea } from '@navikt/ds-react'
+import React, { useState } from 'react'
+import { Klage, Utfall } from '~shared/types/Klage'
+import { KlageVurderingRedigering } from '~components/klage/vurdering/KlageVurderingRedigering'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { oppdaterInitieltUtfallForKlage } from '~shared/api/klage'
+import { isFailureHandler } from '~shared/api/IsFailureHandler'
+import { isPending } from '~shared/api/apiUtils'
+
+export const KlageInitiellVurdering = (props: { klage: Klage }) => {
+  const klage = props.klage
+
+  const [begrunnelse, setBegrunnelse] = useState<string>('')
+  const [utfall, setUtfall] = useState<Utfall>()
+  const [redigerbar, setRedigerbar] = useState<boolean>(!klage?.initieltUtfall)
+
+  const [lagreInitiellStatus, lagreInitiellKlageUtfall] = useApiCall(oppdaterInitieltUtfallForKlage)
+  const harDataFraBackend = true
+
+  const getTextFromutfall = (utfall: Utfall): string => {
+    switch (utfall) {
+      case Utfall.OMGJOERING:
+        return 'Hvorfor skal det omgjøres?'
+      case Utfall.DELVIS_OMGJOERING:
+        return 'Hvorfor skal det delvis omgjøres?'
+      case Utfall.STADFESTE_VEDTAK:
+        return 'Hvorfor står vedtaket seg?'
+    }
+  }
+
+  return (
+    <>
+      {redigerbar && (
+        <>
+          <Heading level="1" size="large">
+            Første vurdering av utfall saksbehandler
+          </Heading>
```

Comment:
Det blir litt off med overskrifter senere i denne siden, siden den også setter level 1 med large. Jeg tenker at vi kan nok ha en level 1 på "Vurder utfall" og så en ekstra level 2 for første vurdering av utfall saksbehandler som et første forslag, men det er nok best å høre med Frida også og høre om hun har noen ideer på totalutformingen av siden

Reply:
Jepp for denne er jo level 1 før vurderingen er gjort så kanskje denne burde være level 1?

Reply:
Jeg tenker litt 

1. Vurder utfall (kan vises uavhengig i toppen av denne komponenten)
   - level 2: Første vurdering av saksbehandler (denne komponenten)
   - level 2: Utfall av klagen (neste komponent)

eller noe i den dur